pyext/src/analysis_trajectories.py

```diff
@@ -42,22 +43,67 @@ class AnalysisTrajectories(object):
     def __init__(self,
                  out_dirs,
                  dir_name = 'run_',
-                 out_name = 'output',
                  analysis_dir = 'analysis',
-                 nproc=6,
-                 nskip=200):
+                 detect_equilibration=True,
+                 burn_in=1000,
+                 nskip=200,
+                 nproc=1,
+                 plot_fmt='pdf'):
+        
+        '''
+        Analyze the ensemble of models obtained after structural sampling by
+        filtering out the bad scoring models, and clustering the good scoring
+        models according to one or multiple score terms, thereby obtaining
+        a set of representative good scoring models that can be used to
+        subsequently compute sampling precision and model precision [using the
+        sampcon module].
+
+        @param out_dirs: list of directories containing the trajectory (rmf
+        files and stat files) for each independent run.
+        
+        @param dir_name: prefix for all directories containing structural 
+        sampling runs. Usually these are named as <prefix>1, <prefix>2, etc.
+        Default is "run_"
+        
+        @analysis_dir: name of output directory where all analysis results will
+        be stored. Default is "analysis".
+        
+        @detect_equilibration: if True, the equilibration period is rigorously
+        determined from the trajectory of all scores using statistical inefficiency measures. Default is True.
+        
+        @burn_in: a total of <burn_in> number of frames are always discarded
+        from the beginning of the trajectory, when calculating statistics.
+        Default is 1000.
+        
+        @nskip: skips every <nskip> frames when calculating statistics.
+        Default is 200.
+        
+        @nproc: number of processors to work on parallely (uses the python multiprocess module). Default is 1.
+        
+        @plot_fmt: file extensions for all plots created. Default is pdf.
+        '''
 
         self.out_dirs = [os.path.abspath(d) for d in out_dirs]
         self.dir_name = dir_name
         self.analysis_dir = os.path.abspath(analysis_dir)
+        self.detect_equilibration = detect_equilibration
         self.nproc = nproc
         self.nskip = nskip
+        self.th = burn_in
+        self.plot_fmt = plot_fmt
```

Comment:
I'd add a catch here for available matplotlib plot formats, otherwise it won't throw an error until after all of the calculations.
```
fig = pl.figure()
available_plot_formats = list(fig.canvas.get_supported_filetypes().keys())
if plot_fmt not in available_plot_formats:
    raise Exception("plot_fmt must be a supported matplotlib file type:", available_plot_formats(
```

Reply:
Godd catch

Reply:
checking for available plot formats now added in commit 871884ee953b0ce282d98cb31373d20bc0b9218e